client/battle/CCreatureAnimation.cpp

```diff
@@ -425,28 +427,22 @@ inline void CCreatureAnimation::putPixel(ui8 * dest, const SDL_Color & color, si
 
 bool CCreatureAnimation::isDead() const
 {
-	return getType() == CCreatureAnim::DEAD
-	    || getType() == CCreatureAnim::DEATH;
+	return getType() == CCreatureAnim::DEAD || getType() == CCreatureAnim::DEATH;
 }
 
 bool CCreatureAnimation::isIdle() const
 {
-	return getType() == CCreatureAnim::HOLDING
-	    || getType() == CCreatureAnim::MOUSEON;
+	return getType() == CCreatureAnim::HOLDING || getType() == CCreatureAnim::MOUSEON;
 }
 
 bool CCreatureAnimation::isMoving() const
 {
-	return getType() == CCreatureAnim::MOVE_START
-	    || getType() == CCreatureAnim::MOVING
-	    || getType() == CCreatureAnim::MOVE_END;
+	return getType() == CCreatureAnim::MOVE_START || getType() == CCreatureAnim::MOVING || getType() == CCreatureAnim::MOVE_END;

```

Comment:
Basically for the reason mentioned in PR description: there is serious issue that Uncrustify try to indent multi-line conditions and align them with spaces. This is controlled following options:
```
# The position of boolean operators in wrapped expressions
pos_bool                        = join     # ignore/join/lead/lead_break/lead_force/trail/trail_break/trail_force
```

Other than that almost all VCMI code that using multi-line condition is just bad or totally horrible: especially CGameHandler with complain and comments inside if-else conditions. It's so bad so in some cases something like that is preferable:
```
bool boolOptionNameOne = somelongcodition == somelongtype;
bool boolOptionNameTwo = somelongcodition == somelongtype && 9000 == 9000;
if(boolOptionNameOne && boolOptionNameTwo)
```
To what we have how:
```
			//ranged counterattack
			if (destinationStack->hasBonusOfType(Bonus::RANGED_RETALIATION)
				&& !stack->hasBonusOfType(Bonus::BLOCKS_RANGED_RETALIATION)
				&& destinationStack->ableToRetaliate()
				&& stack->alive()) //attacker may have died (fire shield)
			{
```